crates/turbopack-ecmascript/src/chunk/mod.rs

```diff
@@ -739,6 +739,21 @@ impl GenerateSourceMap for EcmascriptChunkContent {
     fn generate_source_map(self_vc: EcmascriptChunkContentVc) -> SourceMapVc {
         self_vc.code().generate_source_map()
     }
+
+    #[turbo_tasks::function]
+    async fn by_section(&self, section: StringVc) -> Result<OptionSourceMapVc> {
```

Comment:
You could pass the section by value here as `&str`

Reply:
Done.